cluster/node.go

```diff
@@ -267,6 +269,14 @@ func (n *Node) ReservedCpus() int64 {
 	return r
 }
 
+func (n *Node) OverCommitedMemory() int64 {
```

Comment:
How about simply `AvailableMemory`? Today we are doing overcommitting, tomorrow we may be doing something different.

It might be ambiguous as it might look like "free memory" rather than "how much total memory I can use", but I'd rather not include overcommit in the name.